pkg/tiller/release_modules.go

```diff
@@ -174,7 +173,9 @@ func DeleteRelease(rel *release.Release, vs chartutil.VersionSet, kubeClient env
 			log.Printf("uninstall: Failed deletion of %q: %s", rel.Name, err)
 			if err == kube.ErrNoObjectsVisited {
 				// Rewrite the message from "no objects visited"
-				err = errors.New("object not found, skipping delete")
+				// err = errors.New("object not found, skipping delete")
+				log.Printf("object = %q not found, skipping delete.", rel.Name)
```

Comment:
I think it would be better here to keep using an `error` and use `fmt.Errorf` to get the better error message.

Reply:
If  when the first deletion , some resources is not deleted and some is deleted.
So, when the next deletion , some resources  is not exist and the kube.ErrNoObjectsVisited will happen.
I think that it is the normal condition. 
So, just log this error is ok.